extensions/inventory-management/index.md

```diff
@@ -4,38 +4,49 @@ title: Install Inventory Management
 redirect_from: guides/v2.3/comp-mgr/install-extensions/inventory-management-installation.html
 ---
 
-{{site.data.var.im}} modules provide all inventory features and options for Single and Multi Source merchants to manage product quantities and stock for sales channels. These features are available in Magento 2.3.X Open Source, Commerce, and Commerce Cloud.
+{{site.data.var.im}} modules provide all inventory features and options for Single and Multi Source merchants to manage product quantities and stock for sales channels. These features are available in 2.3.X {{site.data.var.ce}}, {{site.data.var.ee}}, and {{site.data.var.ece}}.
 
 These features and extensions were developed as part of the [Multi Source Inventory (MSI) project](https://github.com/magento-engcom/msi) through the Magento Community Engineering program.
 
-{{site.data.var.im}} installs with Magento Open Source or Commerce with all features enabled by default. No additional steps are required for enabling these inventory features.
+{{site.data.var.im}} installs with Magento {{site.data.var.ce}} and {{site.data.var.ee}} with all features enabled by default. No additional steps are required for enabling these inventory features. Upgrades from v2.1.x or 2.2.x may require additional steps. See [Upgrade Inventory Management](#upgrade-inventory-management).
 
 We recommend installing according to [Install Magento Using Composer]({{site.baseurl}}/guides/v2.3/install-gde/composer.html). You must install with a metapackage to receive all {{site.data.var.im}} modules.
 
-The following line in the `composer.json` metapackage installs Inventory Management:
+The following line in the `composer.json` metapackage installs Inventory Management 1.1.2:
 
 ```json
-        magento/inventory-composer-metapackage = ^1.1.0
+        magento/inventory-composer-metapackage = 1.1.2
 ```
 
+For a list of Inventory Management metapackage versions, see the [release notes]({{site.baseurl}}/guides/v2.3/inventory/release-notes.html).
+
 The {{site.data.var.im}} installation process makes the following changes to the `<Magento_installation_directory>/composer.json` file. A `1` value indicates the corresponding module is enabled.
 
+The following list includes all modules in Inventory Management 1.1.2:
```

Comment:
The code sample below is definitely not json. It's probably php

Reply:
I think it'd be ideal if you listed which modules are new. Also, i'm not sure `Magento_GraphQl` is actually in IM.

(low priority comment--ignore if you want)

Reply:
Will double check the module. Thanks!